bedita-app/models/object_relation.php

```diff
@@ -44,6 +44,31 @@ public function afterFind($results) {
         return $results;
     }
 
+    /**
+     * Escape all arguments that have been passed to this function.
+     * 
+     * @param any[] ...$values Values to be prepared.
+     * @return string[]
+     */
+    private static function prepareAll()
+    {
```

Comment:
[minor] in [BE3 coding styles](https://github.com/bedita/bedita/wiki/Resources-(BEdita-3)#php) opening braces for classes and methos MUST be on the same line.

please could you check and fix all methods that don't follow the guidelines.

Reply:
Done.